lib/services/student_donor.js

```diff
@@ -6,6 +6,10 @@ const { val } = require("objection");
 module.exports = class StudentDonorService extends Schmervice.Service {
   async updateOrInsertById(studentId, donor_id) {
     const { StudentDonor } = this.server.models();
+    if (donor_id.length===0){
+      await StudentDonor.query().where("student_id", studentId).delete()
+      return 1
```

Comment:
@kirithiv2000 Why are you returning 1? Return something meaningful, check other APIs what do they return on success? It should be something like this : `{ success: true }`